components/layout/SideNav.tsx

```diff
@@ -316,36 +419,44 @@ const SideNav = ({ sectionLinks, pages }: SideNavProps) => {
         })}
       >
         <SlidePane>
-          <SectionWrap>
-            {sectionLinks.map((l) => (
-              <SectionLink
-                key={l.href}
-                isActive={l.href === pathname}
-                linkProps={l}
-              />
-            ))}
-          </SectionWrap>
+          {!(activeSection && activeSection.isSection) && (
+            <SectionWrap>
+              {sectionLinks.map((l) => (
+                <SectionLink
+                  key={l.href}
+                  isActive={l.href === pathname}
+                  linkProps={l}
+                />
+              ))}
+            </SectionWrap>
+          )}
+
+          <MobileWrap>
+            <MobileItems {...mobileItems} />
+          </MobileWrap>
         </SlidePane>
 
         <SlidePane key={activeSection?.href}>
-          <SectionWrap>
-            <SectionLink
-              linkProps={{
-                href: "/",
-                IconSvg: BackIcon,
-                isSection: false,
-                text: "Back",
-              }}
-              isActive={false}
-            />
-
-            {activeSection && (
-              <SectionLink
-                linkProps={{ ...activeSection, isSection: false }}
-                isActive
-              />
+          <StickySectionWrap>
+            {activeSection && activeSection.isSection && (
```

Comment:
I found you were actually able to tab over to the links in the hidden pane, so I'm just not rendering the links themselves if the pane isn't active